notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>Remove "The notebook also shows how to join two tables on Trino to create a new table"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>